mods/d2k/tilesets/arrakis.yaml

```diff
@@ -1120,45 +1066,6 @@ Templates:
 		Tiles:
 			0: Rough
 			1: Rock
-	Template@96: # TODO: This should go away
-		Id: 96
-		Image: BLOXBASE
-		Frames: 1,1,1,1,1,1,1,1,1
-		Size: 3,3
-		Category: Rock-Detail
-		PickAny: False
-		Tiles:
-			0: Clear
-			1: Clear
-			2: Clear
-			5: Clear
-			4: Clear
-			3: Clear
-			6: Clear
-			7: Clear
-			8: Clear
-	Template@97: # TODO: This should go away
-		Id: 97
-		Image: BLOXBASE
-		Frames: 1,1,1,1
-		Size: 2,2
-		Category: Rock-Detail
-		PickAny: False
-		Tiles:
-			0: Clear
-			1: Clear
-			3: Clear
-			2: Clear
-	Template@98: # TODO: This should go away
```

Comment:
I assume you removed this because of the comment, but it is a valid tile, so I guess the comment is wrong.